pkg/aws/awscontrolplane/admit_awscontrolplane.go

```diff
@@ -0,0 +1,150 @@
+package awscontrolplane
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/blang/semver"
+	infrastructurev1alpha2 "github.com/giantswarm/apiextensions/pkg/apis/infrastructure/v1alpha2"
+	releasev1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/release/v1alpha1"
+	"github.com/giantswarm/apiextensions/pkg/label"
+	"github.com/giantswarm/backoff"
+	"github.com/giantswarm/k8sclient/v3/pkg/k8sclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"k8s.io/api/admission/v1beta1"
+	"k8s.io/apimachinery/pkg/types"
+	restclient "k8s.io/client-go/rest"
+	apiv1alpha2 "sigs.k8s.io/cluster-api/api/v1alpha2"
+
+	"github.com/giantswarm/admission-controller/pkg/admission"
+	"github.com/giantswarm/admission-controller/pkg/aws"
+)
+
+type Admitter struct {
+	k8sClient              k8sclient.Interface
+	validAvailabilityZones []string
+	logger                 micrologger.Logger
+}
+
+type Config struct {
+	ValidAvailabilityZones string
+	Logger                 micrologger.Logger
+}
+
+// var (
+//  awsControlPlaneResource = metav1.GroupVersionResource{Group: "", Version: "v1", Resource: "awscontrolplane"}
+// )
```

Comment:
What is up with that?

Reply:
I actually don't know. It's a leftover from copypasting the g8scontrolplane admitter and I guess I can remove it